instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/SpringWebfluxTelemetryBuilder.java

```diff
@@ -127,6 +128,25 @@ public SpringWebfluxTelemetryBuilder setCapturedServerResponseHeaders(
     return this;
   }
 
+  /**
+   * Configures the instrumentation to recognize an alternative set of HTTP request methods.
+   *
+   * <p>By default, this instrumentation defines "known" methods as the ones listed in <a
+   * href="https://www.rfc-editor.org/rfc/rfc9110.html#name-methods">RFC9110</a> and the PATCH
+   * method defined in <a href="https://www.rfc-editor.org/rfc/rfc5789.html">RFC5789</a>.
+   *
+   * <p>Note: calling this method <b>overrides</b> the default known method sets completely; it does
+   * not supplement it.
+   *
+   * @param knownMethods A set of recognized HTTP request methods.
+   */
+  @CanIgnoreReturnValue
+  public SpringWebfluxTelemetryBuilder setKnownMethods(Set<String> knownMethods) {
+    clientBuilder.setKnownMethods(knownMethods);
+    httpServerAttributesExtractorBuilder.setKnownMethods(knownMethods);
```

Comment:
Not your issue, but I am noticing the inconsistent naming between client/server fields here.

Reply:
Yeah, that's true -- but it's also because the `clientBuilder` has a completely different type; it's an internal instrumenter builder wrapper.

Reply:
Yeah ok. 🤷🏻